src/atc/etl/extractor.py

```diff
@@ -15,6 +15,7 @@ class Extractor(EtlBase):
     """
 
     def __init__(self, dataset_key: str = None):
+        super().__init__()
```

Comment:
This is maybe a detail, but all these __init__ updates to call the parent constructors. Should these be in its own PR? It does not feel like its related composit-orchestration.

Reply:
they are generally good practice I believe, and do no harm.